packages/cli/src/commands/push.js

```diff
@@ -16,13 +16,14 @@ const register = program => program
   .option('-d, --deploy-libs', 'deploys libraries to the network if there is no existing deployment')
   .option('--reset', 'redeploys all contracts (not only the ones that changed)')
   .option('-f, --force', 'ignores validation errors and deploys contracts')
+  .option('--full-app', 'deploys an App contract to manage the project')
```

Comment:
leaving this comment to remind us that we are thinking of moving this flag to a new `publish` command